lib/src/main/java/coppercore/commands/WaitForButtonCommand.java

```diff
@@ -0,0 +1,32 @@
+package coppercore.commands;
```

Comment:
Do we actually use this? It seems like this would be redundant to wpilib's onTrue function

Reply:
I agree by looking at it, however, I left it in since I figured it was used somewhere. Ill see if i can find any uses of it in 2024 code